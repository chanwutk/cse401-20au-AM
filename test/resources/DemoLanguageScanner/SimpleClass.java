public class SimpleClass {
  public static void main(String args[]) {
    System.out.println(15 + 7 - 2 * args.length);
  }
}
