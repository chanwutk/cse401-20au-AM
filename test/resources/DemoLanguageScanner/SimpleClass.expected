PUBLIC CLASS ID(SimpleClass) LBRACKET PUBLIC STATIC VOID MAIN LPAREN STRING ID(args) LBRACKET_SQUARE RBRACKET_SQUARE RPAREN LBRACKET PRINTLN LPAREN DIGIT(15) PLUS DIGIT(7) MINUS DIGIT(2) RPAREN SEMICOLON RBRACKET RBRACKET